api/v2.go

```diff
@@ -113,7 +113,7 @@ func NewV2Endpoints(c *Client) *V2Endpoints {
 		AlertRules:              &AlertRulesService{c},
 		CloudAccounts:           &CloudAccountsService{c},
 		ContainerRegistries:     &ContainerRegistriesService{c},
-		ResourceGroups:          &ResourceGroupsService{c},
+		ResourceGroups:          NewResourceGroupsVersionService(c),
```

Comment:
I'm not 100% sure this is going to using the schemaService is going to fulfil what you need with this.

For context we used this schemaService when transitioning from the v1 api when cloudAccounts, alertChannels, containerRegistries were under `Integration` resource.  Only Get and Delete are exposed in the `V2Service` interface

Reply:
Yeah I wasn't sure about the changes I actually needed in this file, but once I get to testing out the cli locally, I think I'll understand it better.